### PR TITLE
feat(ui5-select): implement accessibility spec

### PR DESCRIPTION
**Pull request by nnaydenow**
#1287

### CODE REVIEW

File: packages/main/src/Select.js [hunk 1]
```diff
@@ -424,6 +428,29 @@ class Select extends UI5Element {
 		}
 	}
 
+	valueStateTextMappings() {
```
Comment (vladitasev) [inline packages/main/src/Select.js:431]: This should logically be a getter, please change it

File: packages/main/src/Select.hbs [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
 	@keydown="{{_onkeydown}}"
 	@keyup="{{_onkeyup}}"
 	@focusin="{{_onfocusin}}"
 	@focusout="{{_onfocusout}}"
 	@click="{{_toggleRespPopover}}"
 >
-	<div class="ui5-select-label-root">
-		<ui5-label>{{_text}}</ui5-label>
-	</div>
+	<div class="ui5-select-wrapper"
```
Comment (vladitasev) [inline packages/main/src/Select.hbs:10]: Is it really necessary to have one more nested `div` with `width: 100%; height: 100%`?

Comment (elenastoyanovaa) [reply packages/main/src/Select.hbs:10]: Actually, no. This was needed in order to wrap the label and the icon together and add all relevant aria attributes, when the popover was part of the ui5-select. Now this is not needed. 

@nnaydenow you could add aria attributes from the ui5-select-wrapper ui5-select-root. Sorry for not checking that earlier.

File: packages/main/src/Select.js (hunk 1 above)
Comment (nnaydenow) [reply packages/main/src/Select.js]: Done

File: packages/main/src/Select.hbs (hunk 2 above)
Comment (nnaydenow) [reply packages/main/src/Select.hbs]: Done

File: packages/main/src/Select.hbs [hunk 3]
```diff
@@ -1,15 +1,21 @@
...
 	class="ui5-select-root"
-	tabindex="{{tabIndex}}"
 	dir="{{dir}}"
+	tabindex="{{tabIndex}}"
+	id="{{_id}}-select"
+	role="button"
+	aria-haspopup="listbox"
+	aria-labelledby="{{_id}}-label"
+	aria-describedby="{{valueStateTextId}}"
+	?disabled="{{disabled}}"
```
Comment (elenastoyanovaa) [inline packages/main/src/Select.hbs:10]: since this is an ordinary div, not an input , this should be aria-disabled, not just disabled

Comment (nnaydenow) [reply packages/main/src/Select.hbs]: Done

**Pull request merged**